Classes/Issues/Comments/Reactions/ReactionContent+ReactionType.swift

```diff
@@ -19,4 +19,16 @@ extension ReactionContent {
         case .thumbsDown: return "👎"
         }
     }
+    
+    /// User readable name of the current reaction type
+    var name: String {
+        switch self {
+        case .confused: return "confused"
+        case .heart: return "heart"
+        case .hooray: return "horray"
```

Comment:
Typo, `hooray`.

Not sure if this is needed also, but we might want to wrap these in `NSLocalizedString`. Also, I've used `var localizedString` in the `IssueType` (instead of `name`), but I guess that's NBD.